docs/reference/migration/migrate_query_refactoring.asciidoc

```diff
@@ -71,4 +71,5 @@ codebase and avoid duplication.
 Support for `queryName` and `boost` has been streamlined to all of the queries. That is
 a breaking change till queries get sent over the network as serialized json rather
 than in `Streamable` format. In fact whenever additional fields are added to the json
-representation of the query, older nodes might throw error when they find unknown fields.
+representation of the query, older nodes might throw error when they find unknown fields.pd
```

Comment:
Meh. If this had been a wq I would have had an explanation. I'm blaming the fact that focus follows mouse was disabled on my Ubuntu last time I upgraded as a flimsy excuse.